bin/faucet/src/server/get_tokens.rs

```diff
@@ -52,6 +61,12 @@ pub enum InvalidRequest {
     AccountId(#[source] AccountIdError),
     #[error("asset amount {0} is not one of the provided options")]
     AssetAmount(u64),
+    #[error("invalid POW solution")]
+    InvalidPoW,
```

Comment:
Maybe this could include the expected difficulty (and/or received solution, but probably less useful for general debugging since it's arbitrary bytes.)

Reply:
Hm, I think that the only way for it to be useful is to include all of those values, that way we can re-construct the check. But i'm not convinced for it to be that useful really. what do you think?